ibis/backends/duckdb/tests/test_io.py

```diff
@@ -235,52 +235,80 @@ def test_read_sqlite_no_table_name(con, tmp_path):
         scon.close()
 
 
+@pytest.fixture
+def sqlite_path(tmp_path, data_dir):
+    path = tmp_path / "test.db"
+    scon = sqlite3.connect(str(path))
+    try:
+        with scon:
```

Comment:
If you use the context manager here you don't need to explicitly call `.close()`.